test/test_name_anchorages.py

```diff
@@ -2,6 +2,7 @@
 import os
 import yaml
 import pytest
+import six
```

Comment:
@bitsofbits Could be possible to delete the import of six if there is not used?

Reply:
Yep. Done!